OpenTabletDriver.Configurations/Configurations/Parblo/Ninos N7B.json

```
@@ -0,0 +1,52 @@
+{
+  "Name": "Parblo Ninos N7",
```

Tablet name should match the file name.
```suggestion
  "Name": "Parblo Ninos N7B",
```